### PR TITLE
Gcp cloud data prep

### PR DESCRIPTION
**Pull request by Qiwei08**

### CODE REVIEW

File: technologies/job/gcp-cloud-data-prep/gcp-cloud-data-prep/instanceActions.js [hunk 1]
```diff
@@ -0,0 +1,76 @@
...
+
+    const { data } = await axios.get(
+      `${job.featuresValues.endpoint.url}/v4/jobGroups/${instance.payload.jobGroupId}`,
+      getRequestConfigFromEndpointForm(job.featuresValues.endpoint)
+    );
+
+    return Response.success(JOB_STATES[data.status] || JobStatus.AWAITING);
+
+  } catch (error) {
+    console.log(error.data.response)
```
Comment (ntorionbearstudio) [inline technologies/job/gcp-cloud-data-prep/gcp-cloud-data-prep/instanceActions.js:69]: console.log can be removed

File: technologies/job/gcp-cloud-data-prep/package.json [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+  "name": "gcp-cloud-data-prep",
+  "version": "0.1.0",
+  "private": true,
+  "scripts": {
+    "start": "saagie-sdk start",
+    "build": "saagie-sdk build",
+    "new:context": "saagie-sdk init"
+  },
+  "dependencies": {
+    "@google-cloud/storage": "^4.7.0",
```
Comment (ntorionbearstudio) [inline technologies/job/gcp-cloud-data-prep/package.json:11]: Package can be removed in `package.json`

File: technologies/job/gcp-cloud-data-prep/errors.js [hunk 3]
```diff
@@ -0,0 +1,10 @@
+export const ERRORS_MESSAGES = {
+    LOGIN_ERROR: 'Login error, please check your credentials in Endpoint form',
+    MISSING_RUN_ENV_ERROR: 'Failed to start job from Dataprep, missing running environment informations',
+    MISSING_RUN_ENV_FULL_ERROR: 'Failed to start job from Dataprep: please check the Wrangled Dataset ID',
+    FAILED_TO_RUN_JOB_ERROR: 'Failed to start job from Dataprep',
+    FAILED_TO_GET_STATUS_ERROR: 'Failed to get status for job from Dataprep',
+  }
+  
+  export const VALIDATION_FIELD = 'ValidationFailed';
+  export const SSL_ISSUES_CODE = 'UNABLE_TO_VERIFY_LEAF_SIGNATURE';
```
Comment (ntorionbearstudio) [inline technologies/job/gcp-cloud-data-prep/errors.js:10]: This constant can be removed

File: technologies/job/gcp-cloud-data-prep/gcp-cloud-data-prep/instanceActions.js [hunk 4]
```diff
@@ -0,0 +1,76 @@
...
+        wrangledDataset: {
+          id: parseInt(job.featuresValues.datasetID),
+        }
+      },
+      getRequestConfigFromEndpointForm(job.featuresValues.endpoint)
+    );
+    // You can return any payload you want to get in the stop and getStatus functions.
+    return Response.success({ jobGroupId: data.id });
+  } catch (error) {
+    console.log(error.response.data)
```
Comment (ntorionbearstudio) [inline technologies/job/gcp-cloud-data-prep/gcp-cloud-data-prep/instanceActions.js:29]: console.log can be removed

File: technologies/job/gcp-cloud-data-prep/gcp-cloud-data-prep/instanceActions.js (hunk 1 above)
Comment (Qiwei08) [reply technologies/job/gcp-cloud-data-prep/gcp-cloud-data-prep/instanceActions.js]: Ok, it's removed in the last commit

**Pull request merged**